topi/python/topi/cuda/nms.py

```diff
@@ -351,6 +358,9 @@ def non_max_suppression(data, valid_count, max_output_size=-1,
     valid_count : tvm.te.Tensor
         1-D tensor for valid number of boxes.
 
+    indices : tvm.te.Tensor
+        2-D tensor with shape [batch_size, num_anchors].
```

Comment:
Please be more specific of what `indices` is.

Reply:
have updated in commit [5b5dda5](https://github.com/apache/incubator-tvm/pull/4312/commits/5b5dda5389171c934b94b6b4ef664caf74488b8e)